parts/plugins/x-php.py

```diff
@@ -59,6 +59,7 @@ def schema(cls):
         return schema
 
     def __init__(self, name, options, project):
+        options.configflags.extend(['--with-libdir=lib/{}'.format(project.arch_triplet)])
```

Comment:
Huh, okay. Another question: this is adding `--with-libdir` to the build of PHP itself, when the examples I see using it are on the build of individual PHP extensions. Is this flag in the wrong spot?